modules/oe_content_project/config/install/field.field.node.oe_project.oe_project_budget.yml

```diff
@@ -0,0 +1,22 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.oe_project_budget
+    - node.type.oe_project
+id: node.oe_project.oe_project_budget
+field_name: oe_project_budget
+entity_type: node
+bundle: oe_project
+label: 'Overall budget'
+description: ''
+required: false
+translatable: false
+default_value: {  }
+default_value_callback: ''
+settings:
+  min: null
+  max: null
+  prefix: €
+  suffix: ''
+field_type: float
```

Comment:
I know that mostly budgets are round numbers, but I did not find any requirements in the specification regarding this, so business should know about the fact that in the float field their 90009 values will be rounded up. If that's correct, please ignore my comment.

Reply:
This is correctly set to float, as per COMM requirements.